style/CONTRIBUTE.md

```diff
@@ -248,3 +248,25 @@ Additionally, **`refsByWayName`** is an object mapping way names to text that ca
 - Each route would be recognizable by an initialism, even though it is not signposted.
```

Comment:
This list is redundant to the explanation about Kentucky parkways below. Should we keep it?